index.js

```diff
@@ -45,11 +34,9 @@ const tryReadFile = {
 
 const tryExecFile = {
   async (file, args, options) {
-    return new Promise(resolve => {
-      cp.execFile(file, args, options, (err, stdout) => {
-        resolve(err ? null : stdout)
-      })
-    })
+    return promisify(cp.execFile)(file, args, options)
```

Comment:
Is there any point in storing the promisified function?

Reply:
It would be a tiny optimization.